### DISCUSSION TITLE
Yalancılar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#68] [Zeka Sorusu] [answered]
Ahu isimli kişi Pazartesi, Salı ve Çarşamba günleri; Bilal isimli kişi ise Perşembe, Cuma ve Cumartesi günleri yalan, diğer günlerde doğru söylüyor. Diyelim ki bir gün bu kişilerle yolda karşılaştınız ve bu kişilerden biri aşağıdaki cümlelerden birini, öbür kişi de diğer cümleyi söyledi. Buna göre bu kişilerden hangisinin kim olduğunu ve karşılaştığınız günün hangi gün olduğunu bulabilir misiniz?
X: Yarın yalan söyleyeceğim.
Y: Dün yalan söyledim, yarın da yalan söyleyeceğim.

**Comment by Eceg25** [comment, marked as answer]
Ahu Pazartesi, Salı ve Çarşamba günü kesinlikle yalan söylemekte ama diğer günler doğru mu yalan mı söylediği belli değildir.
Bilal ise Perşembe, Cuma ve Cumartesi yalan, bunların dışındaki günlerde doğru söylemektedir.
Buna göre Çarşamba günü karşılaşmışlardır.
X Bilal, Y ise Ahu'dur.

**Comment by TheEmirAK** [comment]
x=Bilal
Y=Ahu
Çarşamba

**Comment by Onuralp000** [comment]
Ahu Pazartesi, Salı ve Çarşamba günleri doğru söyleyip, Bilal ise Perşembe, Cuma ve Cumartesi günleri doğru söyleyip, karşılaştığımız gün Cuma'dır.
